### ISSUE TITLE
CStringArray derive [CReprOf, CDrop] error

### ISSUE THREAD

**Issue by curiousmindflow** [snipsco/snips-utils-rs#48]
Hello,

I'm encountering issue when deriving CreprOf and CDrop on a struct that have a field of type '*const CStringArray'
Like this:

```Rust
#[derive(Debug, Serialize, Deserialize, Clone)]
pub struct TestStringArray {
    s: Vec<String>,
}

#[derive(Debug, CReprOf, AsRust, CDrop, RawPointerConverter, Clone)]
#[repr(C)]
#[target_type(TestStringArray)]
pub struct TestStringArray_t {
    s: *const CStringArray,
}

#[no_mangle]
pub extern "C" fn nodus__msg__TestStringArray_t__create(s: *const CStringArray) -> *mut c_void {
    let s = unsafe { CStringArray::as_rust(&*s as &CStringArray).unwrap() };

    let msg = TestStringArray { s };
    let msg = TestStringArray_t::c_repr_of(msg).unwrap();
    Box::into_raw(Box::new(msg)) as *mut c_void
}

#[no_mangle]
pub extern "C" fn nodus__msg__TestStringArray_t__destroy(msg: *mut TestStringArray_t) -> () {
    let msg = unsafe { Box::from_raw(msg) };
}
```
I'm facing these issues:

```shell
error[E0599]: no method named `into_raw_pointer` found for struct `CStringArray` in the current scope
  --> nodus-bindings/src/generated/msgs/TestStringArray.rs:18:17
   |
18 | #[derive(Debug, CReprOf, AsRust, CDrop, RawPointerConverter, Clone)]
   |                 ^^^^^^^ method not found in `CStringArray`
   |
   = note: this error originates in the derive macro `CReprOf` (in Nightly builds, run with -Z macro-backtrace for more info)
help: one of the expressions' fields has a method of the same name
   |
18 | #[derive(Debug, size.CReprOf, AsRust, CDrop, RawPointerConverter, Clone)]
   |                 +++++

error[E0599]: no function or associated item named `drop_raw_pointer` found for struct `CStringArray` in the current scope
  --> nodus-bindings/src/generated/msgs/TestStringArray.rs:18:34
   |
18 | #[derive(Debug, CReprOf, AsRust, CDrop, RawPointerConverter, Clone)]
   |                                  ^^^^^ function or associated item not found in `CStringArray`
   |
   = note: this error originates in the derive macro `CDrop` (in Nightly builds, run with -Z macro-backtrace for more info)

For more information about this error, try `rustc --explain E0599`.
error: could not compile `nodus-bindings` due to 2 previous errors
```

What am i doing wrong ? Could be bug ?

**Comment by fredszaq** [comment]
Hello @curiousmindflow, PR #49 should fix this, we were missing the impl of `RawPointerConverter` for `CStringArray`... this should fix it, can you test it ? I'll release a 0.6.1 with the fix once you've validated it on your side

**Comment by fredszaq** [comment]
fixed in 0.6.1
